src/main/java/org/humancellatlas/ingest/project/Project.java

```diff
@@ -65,6 +65,15 @@ public SubmissionEnvelope getOpenSubmissionEnvelope(){
         return null;
     }
 
+    public Boolean getHasOpenSubmission(){
+        for (SubmissionEnvelope submissionEnvelope : this.submissionEnvelopes) {
+            if (submissionEnvelope.isOpen()){
+                return true;
+            }
+        }
+        return false;
+    }
+
     @JsonIgnore
```

Comment:
not sure of the use this annotation here. It is normally used on properties or getters but not methods.

Reply:
It can be used both for properties and methods https://fasterxml.github.io/jackson-annotations/javadoc/2.5/com/fasterxml/jackson/annotation/JsonIgnore.html